### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by nerdstein**
1. Beta upgrades
2. Code refactoring comments by @EclipseGc and dawehner
3. Automated test cleanup

### CODE REVIEW

**Comment by nerdstein** [comment]
Ok tests are finally passing

File: src/KeyProviderBase.php [hunk 1]
```diff
@@ -8,10 +8,8 @@
 
 use Drupal\Component\Plugin\PluginBase;
```
Comment (EclipseGc) [inline src/KeyProviderBase.php]: Need to change this to \Drupal\Component\Plugin\PluginBase

File: src/Plugin/KeyProvider/ConfigKeyProvider.php [hunk 2]
```diff
@@ -38,8 +38,8 @@ public function defaultConfiguration() {
   public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
     $form['key_value'] = array(
       '#type' => 'textarea',
-      '#title' => $this->t('Key value'),
-      '#description' => $this->t("Enter the key to save in Drupal's configuration system."),
+      '#title' => t('Key value'),
```
Comment (EclipseGc) [inline src/Plugin/KeyProvider/ConfigKeyProvider.php]: so this should still be $this->t(). Extending \Drupal\Core\Plugin\PluginBase, instead of Component, will provide the t() method.

Comment (nerdstein) [inline src/Plugin/KeyProvider/ConfigKeyProvider.php]: I tried that and the class extends PluginBase. What am I missing?

Comment (nerdstein) [inline src/Plugin/KeyProvider/ConfigKeyProvider.php]: Let me rephrase - KeyProviderBase extends PluginBase. Wouldn't that get it?
